include/zephyr/bluetooth/hci.h

```diff
@@ -113,6 +113,16 @@ int bt_hci_cmd_send_sync(uint16_t opcode, struct net_buf *buf,
  */
 int bt_hci_get_conn_handle(const struct bt_conn *conn, uint16_t *conn_handle);
 
+/** @brief Get connection given a connection handle.
+ *
+ * @param handle The connection handle
+ * @param type   The connection handle type
+ *
+ * @retval The corresponding connection object on success,
+ *         NULL if it does not exist.
```

Comment:
```suggestion
 * @retval The corresponding connection object on success
 * @retval NULL if it does not exist
```

Isn't it this the proper way of using `@retval`? One line for each return value? Or does the current version also work?

Reply:
`@retval` renders as a table. It's most useful for enumerating values or ranges. It will look incorrect, like this:

> Return values:
> <table>
> <tr><td><code>The<td>corresponding connection object on success,
> <tr><td><code>NULL<td>if it does not exist
> </table>

The alternative is `@returns`:

```suggestion
 * @returns The corresponding connection object on success,
 *         NULL if it does not exist.
```

Or to add in a placeholder like this:

```suggestion
 * @retval NULL if it does not exist.
 * @retval other The corresponding connection object on success,
```